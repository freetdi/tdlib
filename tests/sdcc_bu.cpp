
#define GDIR boost::bidirectionalS
#define TDIR boost::undirectedS
#define NO_NICIFY
#include "sdcc_.cpp"
