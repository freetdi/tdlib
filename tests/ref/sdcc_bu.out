==test graph==
0 --> 1 13 7 8 
1 --> 14 2 9 
2 --> 10 3 8 
3 --> 15 4 9 
4 --> 10 11 5 
5 --> 12 14 6 
6 --> 11 13 7 
7 --> 12 15 
8 --> 12 14 
9 --> 11 13 16 
10 --> 12 15 
11 --> 14 
12 --> 
13 --> 15 
14 --> 
15 --> 
16 --> 
17 --> 
18 --> 
19 --> 
20 --> 
====
PP+FI+TM
0 <--> 8 17 18 19 20 
1 <--> 7 
2 <--> 8 
3 <--> 7 16 
4 <--> 7 
5 <--> 7 
6 <--> 8 
7 <--> 1 3 4 5 8 
8 <--> 0 2 6 7 9 
9 <--> 8 10 
10 <--> 9 11 
11 <--> 10 12 
12 <--> 11 13 
13 <--> 12 14 
14 <--> 13 15 
15 <--> 14 
16 <--> 3 
17 <--> 0 
18 <--> 0 
19 <--> 0 
20 <--> 0 
8
0: 3 7 10 13 15
1: 1 5 8 11 14
2: 5 7 8 10 12
3: 1 3 9 11 13
4: 5 6 7 11 13
5: 3 4 5 10 11
6: 1 2 3 8 10
7: 1 3 5 7 8 10 11 13
8: 1 3 5 7 8 10 13
9: 1 3 5 7 8 13
10: 1 3 7 8 13
11: 0 1 7 8 13
12: 0 1 7 8
13: 0 1 7
14: 0 1
15: 0
16: 9 16
17: 17
18: 18
19: 19
20: 20
incomplete ../../tests/sdcc_.cpp:105:do_it
FI
0 <--> 20 
1 <--> 20 
2 <--> 20 
3 <--> 20 
4 <--> 5 
5 <--> 4 16 
6 <--> 15 
7 <--> 12 
8 <--> 12 
9 <--> 12 
10 <--> 14 
11 <--> 12 
12 <--> 7 8 9 11 14 
13 <--> 14 
14 <--> 10 12 13 15 
15 <--> 6 14 16 
16 <--> 5 15 17 
17 <--> 16 18 
18 <--> 17 19 
19 <--> 18 20 
20 <--> 0 1 2 3 19 
8
0: 17
1: 18
2: 19
3: 20
4: 9 16
5: 1 3 9 11 13
6: 5 6 7 11 13
7: 3 7 10 13 15
8: 3 4 5 10 11
9: 5 7 8 10 12
10: 0 1 7 8 13
11: 1 2 3 8 10
12: 1 3 5 7 8 10 11 13
13: 1 5 8 11 14
14: 1 3 5 7 8 11 13
15: 1 3 5 7 11 13
16: 1 3 7 11 13
17: 1 3 7 13
18: 3 7 13
19: 3 13
20: 13
incomplete ../../tests/sdcc_.cpp:105:do_it
thorup
0 <--> 1 
1 <--> 0 2 
2 <--> 1 3 
3 <--> 2 4 
4 <--> 3 5 
5 <--> 4 6 
6 <--> 5 7 
7 <--> 6 8 
8 <--> 7 9 
9 <--> 8 10 
10 <--> 9 11 
11 <--> 10 12 
12 <--> 11 13 
13 <--> 12 14 
14 <--> 13 15 
15 <--> 14 16 
16 <--> 15 17 
17 <--> 16 18 
18 <--> 17 19 
19 <--> 18 20 
20 <--> 19 21 
21 <--> 20 
11
0:
1: 20
2: 19
3: 18
4: 17
5: 16
6: 0 16
7: 0 15 16
8: 0 10 15 16
9: 0 10 14 15 16
10: 0 10 13 14 15 16
11: 0 10 12 13 14 15 16
12: 0 10 11 12 13 14 15 16
13: 0 9 10 11 12 13 14 15 16
14: 0 8 9 10 11 12 13 14 15
15: 0 7 8 9 10 11 12 13 14 15
16: 0 6 7 8 9 10 11 12 13 14 15
17: 0 5 6 8 9 10 11 12 14 15
18: 0 4 5 8 9 10 11 14 15
19: 0 3 4 8 9 10 14 15
20: 0 2 3 8 9 10 14
21: 0 1 2 9 14
incomplete ../../tests/sdcc_.cpp:105:do_it
FI
0 <--> 20 
1 <--> 20 
2 <--> 20 
3 <--> 20 
4 <--> 5 
5 <--> 4 16 
6 <--> 15 
7 <--> 12 
8 <--> 12 
9 <--> 12 
10 <--> 14 
11 <--> 12 
12 <--> 7 8 9 11 14 
13 <--> 14 
14 <--> 10 12 13 15 
15 <--> 6 14 16 
16 <--> 5 15 17 
17 <--> 16 18 
18 <--> 17 19 
19 <--> 18 20 
20 <--> 0 1 2 3 19 
8
0: 17
1: 18
2: 19
3: 20
4: 9 16
5: 1 3 9 11 13
6: 5 6 7 11 13
7: 3 7 10 13 15
8: 3 4 5 10 11
9: 5 7 8 10 12
10: 0 1 7 8 13
11: 1 2 3 8 10
12: 1 3 5 7 8 10 11 13
13: 1 5 8 11 14
14: 1 3 5 7 8 11 13
15: 1 3 5 7 11 13
16: 1 3 7 11 13
17: 1 3 7 13
18: 3 7 13
19: 3 13
20: 13
incomplete ../../tests/sdcc_.cpp:105:do_it
PP+MD
0 <--> 8 17 18 19 20 
1 <--> 9 
2 <--> 10 
3 <--> 8 16 
4 <--> 8 
5 <--> 9 
6 <--> 10 
7 <--> 8 
8 <--> 0 3 4 7 9 
9 <--> 1 5 8 10 
10 <--> 2 6 9 11 
11 <--> 10 12 
12 <--> 11 13 
13 <--> 12 14 
14 <--> 13 15 
15 <--> 14 
16 <--> 3 
17 <--> 0 
18 <--> 0 
19 <--> 0 
20 <--> 0 
8
0: 3 7 10 13 15
1: 1 5 8 11 14
2: 5 7 8 10 12
3: 1 3 9 11 13
4: 5 6 7 11 13
5: 3 4 5 10 11
6: 1 2 3 8 10
7: 0 1 7 8 13
8: 1 3 5 7 8 10 11 13
9: 1 3 5 7 8 10 11
10: 1 3 5 7 8 10
11: 1 3 5 7 8
12: 1 3 5 7
13: 1 3 5
14: 1 3
15: 1
16: 9 16
17: 17
18: 18
19: 19
20: 20
incomplete ../../tests/sdcc_.cpp:105:do_it
ppta
incomplete ./treedec/exact_base.hpp:335:try_it
incomplete ./treedec/bits/trie.hpp:336:TRIE
0 <--> 1 11 12 13 14 
1 <--> 2 0 10 
2 <--> 1 3 4 5 
3 <--> 2 
4 <--> 2 
5 <--> 2 6 7 8 9 
6 <--> 5 
7 <--> 5 
8 <--> 5 
9 <--> 5 
10 <--> 1 
11 <--> 0 
12 <--> 0 
13 <--> 0 
14 <--> 0 
7
0:
1: 4 5 6 9 11 12 14
2: 0 4 6 9 12 14 15
3: 0 6 9 13 15
4: 0 6 7 12 15
5: 0 2 4 9 12 14 15
6: 2 4 10 12 15
7: 0 2 8 12 14
8: 2 3 4 9 15
9: 0 1 2 9 14
10: 9 16
11: 17
12: 18
13: 19
14: 20
incomplete ../../tests/sdcc_.cpp:105:do_it
PP+FI
0 <--> 16 18 19 20 21 
1 <--> 8 
2 <--> 10 
3 <--> 8 
4 <--> 8 
5 <--> 10 17 
6 <--> 11 
7 <--> 8 
8 <--> 1 3 4 7 10 
9 <--> 10 
10 <--> 2 5 8 9 11 
11 <--> 6 10 12 
12 <--> 11 13 
13 <--> 12 14 
14 <--> 13 15 
15 <--> 14 16 
16 <--> 15 0 
17 <--> 5 
18 <--> 0 
19 <--> 0 
20 <--> 0 
21 <--> 0 
8
0:
1: 3 7 10 13 15
2: 5 6 7 11 13
3: 5 7 8 10 12
4: 3 4 5 10 11
5: 1 3 9 11 13
6: 0 1 7 8 13
7: 1 2 3 8 10
8: 1 3 5 7 8 10 11 13
9: 1 5 8 11 14
10: 1 3 5 7 8 11 13
11: 1 3 5 7 8 13
12: 1 3 5 7 13
13: 1 3 7 13
14: 3 7 13
15: 3 7
16: 3
17: 9 16
18: 17
19: 18
20: 19
21: 20
incomplete ../../tests/sdcc_.cpp:105:do_it
